docs/dev/setup.rst

```diff
@@ -77,6 +77,37 @@ The app should now be available at http://localhost:8000 or the custom SITE_URL.
 
 And with that, you're ready to start :doc:`contributing`!
 
+Writing to external repositories
```

Comment:
In order to keep the setup instructions short and simple, I would move this section to the CONTRIBUTING file, which also documents how to restore DB backups, which seems to be from the same mold.

Reply:
Let me know if that's the right place in the doc.